auth/oidc.go

```diff
@@ -439,3 +446,23 @@ func SetURLQueryParam(strURL, name, value string) (string, error) {
 	uri.RawQuery = rawQuery.Encode()
 	return uri.String(), nil
 }
+
+// GetHttpClient returns a new HTTP client with TLS certificate verification
+// disabled when insecureSkipVerify is true and enabled otherwise.
+func GetHttpClient(insecureSkipVerify bool) *http.Client {
+	if insecureSkipVerify {
+		transport := &http.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		}
+		return &http.Client{Transport: transport}
+	}
+	return &http.Client{}
+}
+
+// GetClientContext returns a new Context that carries the provided HTTP client
+// with TLS certificate verification disabled when insecureSkipVerify is true and
+// enabled otherwise.
+func GetClientContext(insecureSkipVerify bool) context.Context {
```

Comment:
Should be renamed to something OIDC-specific.

```suggestion
func GetOIDCClientContext(insecureSkipVerify bool) context.Context {
```

Reply:
GetClientContext has been renamed to GetOIDCClientContext.